go/stellar/transform.go

```diff
@@ -212,6 +212,16 @@ func transformPaymentRelay(mctx libkb.MetaContext, acctID stellar1.AccountID, p
 		loc.ToType = stellar1.ParticipantType_STELLAR
 		loc.ToUsername = ""
 		loc.ToAccountName = ""
+		if loc.StatusSimplified == stellar1.PaymentStatus_CANCELED {
```

Comment:
would slightly prefer `p.Claim.TxStatus == stellar1.TransactionStatus_SUCCESS` so the ordering of assignments doesn't matter